hack/e2e-test-ocp.sh

```diff
@@ -8,7 +8,7 @@ set -o pipefail
 export OC=$(which oc) # OpenShift CLI
 SOURCE_DIR="$(cd "$(dirname -- "${BASH_SOURCE[0]}")" && pwd -P)"
 ROOT_DIR="$SOURCE_DIR/.."
-
+INITIAL_NAMESPACE="kueue-system"
```

Comment:
Perhaps DEFAULT_NAMESPACE?

Reply:
Yeah, actually I took this pattern from the e2e-common.sh where they use that for an image. Let me address this.